examples/interpreter-in-browser/www/chain/run.erl

```diff
@@ -0,0 +1,5 @@
+-module(run).
+
+run() ->
+    lumen_intrinsics:println(running_chain),
+    'Elixir.Chain':dom(100).
```

Comment:
Not that it has to be in this PR since you've been working on this PR for awhile, but will my `Executor`, and `js_sys::Promise` system I used in the hand-compiled work for the interpreter so we can vary this value?